templates/rsyslog.conf.erb

```diff
@@ -67,6 +67,14 @@ $Umask <%= scope.lookupvar('rsyslog::umask') %>
 $OmitLocalLogging on
 <% end -%>
 
+# Settings for imjournal
+<% if scope.lookupvar('rsyslog::im_journal_statefile') -%>
+$IMJournalStateFile <%=scope.lookupvar('rsyslog::im_journal_statefile') %>
+<% end %>
+$ImjournalIgnorePreviousMessages <%=scope.lookupvar('rsyslog::im_journal_ignore_previous_messages') %>
```

Comment:
Awesome :) I'm not a maintainer at all, so won't be able to merge the PR - perhaps when @saz or another maintainer has a chance, this PR could be merged in.